colcon_cmake/task/cmake/build.py

```diff
@@ -325,13 +325,10 @@ def _get_make_arguments(self):
             cmd += ['--install', args.build_base]
         else:
             # Fallback to building the install target.
-            #
-            # Log an error if we are here because we failed to get a meaningful
-            # version number rather than simply using an unsupported version.
             if not cmake_ver:
-                logger.error(
-                    'Unable to determin CMake version. Fallback to building '
-                    'the install target in place of cmake --install.')
+                logger.warning(
```

Comment:
Can you please clarify why `not cmake_ver` would be `True` and this warning be triggered if a project has no install target?

Reply:
I will be `False` on failure to parse rather than `None`. This is so the upstream doce can distuguish between no parsing attempt made (`None`) and failed to parse (`False`).